db/migrate/20231026011717_create_parachutes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateParachutes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :parachutes do |t|
+      t.string :name
+      t.string :city
+
+      t.timestamps
+    end
+  end
```

Comment:
It is good to have the attributes `name` and `city` for the parachutes table. But we need more details to convince end users. so please kindly add `rent` and `duration` attributes too.